jujugui/static/gui/src/app/components/added-services-list-item/test-added-services-list-item.js

```diff
@@ -435,22 +445,13 @@ describe('AddedServicesListItem', function() {
       <juju.components.SvgIcon name="focused_16" size="16"/>);
     assert.deepEqual(
       output.props.children[4].props.children[1].props.children,
-      <juju.components.SvgIcon name="hide_16" size="16"/>);
+      <juju.components.SvgIcon name="show_16" size="16"/>);
```

Comment:
No, but I changed the test slightly (see lines 427 and 428) to test one toggle (focus/highlight) on true and the other (fade) on false.